tests/builds/zika.t

```diff
@@ -86,7 +86,7 @@ Build a time tree from the existing tree topology, the multiple sequence alignme
   >  --date-confidence \
   >  --date-inference marginal \
   >  --clock-filter-iqd 4 \
-  >  --seed 314159 > /dev/null
+  >  --seed 314159 &> /dev/null
```

Comment:
I pulled this out as a separate PR #1123, since it's independent and we could benefit from it separately. Should've done this sooner!